realm/realm-library/src/objectServer/java/io/realm/SyncManager.java

```diff
@@ -230,6 +232,25 @@ private static synchronized void removeSession(SyncConfiguration syncConfigurati
         }
     }
 
+    /**
+     * Retruns the all valid sessions belonging to the user.
```

Comment:
I found that @cmelchior sent a DM to me at July 20th as follows:

> I don’t think using the Object Store `SyncUser::all_sessions()` is worth it. We track all of them in Java already.

How about using just `sessions` in this PR and discuss it later?

PS: I should modify this PR's implementation to exclude error state sessions.